### PR TITLE
model updates

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
**Are you a current Fivetran customer?** 
Fivetran PR.

**What change(s) does this PR introduce?** 

End model enhancements.

**Did you update the CHANGELOG?** 

- [ ] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [ ] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [ ] Yes, Issue/Feature [link bug/feature number here]
- [ ] No 

**How did you test the PR changes?** 

 

- [x] Buildkite  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [ ] BigQuery
- [ ] Redshift
- [ ] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: models/shopify__orders.sql [hunk 1]
```diff
@@ -54,6 +54,39 @@ with orders as (
...
+    
+    from {{ var('shopify_order_url_tag') }}
+    group by 1,2
+
+), fulfillments as (
+
+    select 
+        order_id,
+        source_relation,
+        count(order_id) as number_of_fulfillments,
```
Comment (fivetran-jamie) [inline models/shopify__orders.sql:82]:
```suggestion
        count(distinct fulfillment_id) as number_of_fulfillments,
```

**Review by fivetran-jamie — COMMENTED** [review]
looks great!!! just 3 tiny suggestions

File: models/intermediate/int_shopify__discounts__order_aggregates.sql [hunk 2]
```diff
@@ -18,11 +18,11 @@ orders_aggregated as (
...
-        sum(orders.total_line_items_price) as total_order_line_items_price,
-        sum(orders.shipping_cost) as total_order_shipping_cost,
-        sum(orders.refund_subtotal + orders.refund_total_tax) as total_order_refund_amount,
-        count(distinct customer_id) as count_customers,
-        count(distinct email) as count_customer_emails,
+        max(orders.total_line_items_price) as total_order_line_items_price, -- summing would multiply the total by the # of discount codes applied to an order
+        max(orders.shipping_cost) as total_order_shipping_cost, -- summing would multiply the total by the # of discount codes applied to an order
+        max(orders.refund_subtotal + orders.refund_total_tax) as total_order_refund_amount, -- summing would multiply the total by the # of discount codes applied to an order
+        count(distinct customer_id) as count_distinct_customers,
+        count(distinct email) as count_distinct_customer_emails,
```
[the remark this answers is not in the document]
Comment (fivetran-catfritz) [reply models/intermediate/int_shopify__discounts__order_aggregates.sql]: thanks updated!

File: models/shopify__discounts.sql [hunk 3]
```diff
@@ -67,8 +67,8 @@ aggregates_joined as (
         coalesce(orders_aggregated.total_order_shipping_cost, 0) as total_order_shipping_cost,
         coalesce(abandoned_checkouts_aggregated.total_abandoned_checkout_shipping_price, 0) as total_abandoned_checkout_shipping_price,
         coalesce(orders_aggregated.total_order_refund_amount, 0) as total_order_refund_amount,
-        coalesce(orders_aggregated.count_customers, 0) as count_customers,
-        coalesce(orders_aggregated.count_customer_emails, 0) as count_customer_emails,
+        coalesce(orders_aggregated.count_distinct_customers, 0) as count_distinct_customers,
+        coalesce(orders_aggregated.count_distinct_customer_emails, 0) as count_distinct_customer_emails,
```
[the remark this answers is not in the document]
Comment (fivetran-catfritz) [reply models/shopify__discounts.sql]: updated!

File: models/shopify__orders.sql (hunk 1 above)
Comment (fivetran-catfritz) [reply models/shopify__orders.sql]: Makes sense. Updated!

**Review by fivetran-jamie — APPROVED** [review]
good to merge once databricks passes!

**Pull request merged**